<launch>
    <!-- launching robot Description and gazebo -->
    <include file="$(find rover_description)/launch/rover_description.launch">
        <arg name="world_file" value="worlds/moon_surface.world"/>
        <!--world should be saved in "usr/share/gazebo-7/worlds"-->
        <!-- <arg name="world_file" value="worlds/cube.world"/> -->
        <arg name="gui_state" value="false"/>
        <arg name="enable_manual_cmd_vel" value="false"/>
    </include>

    <!-- Mapping launch file -->
    <include file="$(find rover_nav)/launch/octo_mapping.launch"/>
    
    <rosparam file="$(find rover_nav)/config/grid_map/filters.yaml" command="load" />
    
    <!-- planner tuner node -->
    <node pkg="rover_nav" type="planner_tuner" name="planner_tuner"/>
    
    <!-- rqt_reconfigure node GUI -->
    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find rover_nav)/rviz/planner_tunning.rviz"/>
</launch>